apiary-metastore-events/kafka-metastore-events/kafka-metastore-listener/src/test/java/com/expediagroup/apiary/extensions/events/metastore/kafka/listener/ListenerUtilsTest.java

```diff
@@ -51,7 +53,7 @@ public void success() {
     CodahaleMetrics metrics = (CodahaleMetrics) MetricsFactory.getInstance();
     Counter counter = metrics.getMetricRegistry()
       .counter(MetricsConstant.LISTENER_SUCCESSES);
-    assertThat(counter.getCount()).isEqualTo(1L);
+    await().atMost(Durations.FIVE_SECONDS).until(() -> counter.getCount() == 1L);
```

Comment:
Ran into some issues with this test where the count wasn't always getting set immediately. added a small await to account for those situations

Reply:
OK